packages/perspective-jupyterlab/test/jupyter/utils.js

```diff
@@ -112,17 +121,49 @@ module.exports = {
         // wait for a cell to be active
         await page.waitForSelector(
             '.jp-Notebook-ExecutionIndicator:not([data-status="idle"])',
-            {timeout: 3000}
+            {timeout: 5000}
         );
 
         await page.waitForSelector(
             '.jp-Notebook-ExecutionIndicator[data-status="idle"]',
-            {timeout: 3000}
+            {timeout: 10000}
```

Comment:
These timeouts are really long and there is no explanation as to why they were added (a total of 45s per test added in this section, 30000, 10000, 5000).  Does that mean the new tests were flakey and this additional tolerance fixes it?  If so, this is a strong indication to me that the conditions we are awaiting are flawed.

Otherwise, if these are just debug code that snuck into the PR, please revert

Reply:
Yes these needed to be extended to avoid spurious failures. This code is basically waiting for the python kernel to process the initial perspective object instantiation, the timing of which is going to vary based on the specs and load of the CI machine. I don't know how you would configure any wait to be long enough to account for these variations without introducing flakiness while being short enough to not extend the build too long during failed builds.

Theoretically these waits should be as long as possible, and we should have jest fast-fail since if you're waiting too long it's highly likely your test is failing.